databases/week1/db-week1-hw.sql

```diff
@@ -0,0 +1,58 @@
+1. Find out how many tasks are in the task table
+SELECT COUNT(*) FROM task;
+
+2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(*) FROM task WHERE due_date IS NULL;
+
+
+3. Find all the tasks that are marked as done
+SELECT task.id, task.title, status.name, status.id FROM task, status where status.name LIKE "Done" 
+AND task.status_id=status.id;
+
+using Join
+SELECT t.id,t.title,t.description,t.status_id,t.user_id 
```

Comment:
Why not `SELECT*` here as you selected several columns or `SELECT task.title, status.name from .....`